Practical Example/RW_interview_prep2/RW_interview_prep/Model/DataModel.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  DataModel.swift
+//  RW-takeHomeProject
+//
+//  Created by Duc Dang on 8/25/20.
+//  Copyright © 2020 Duc Dang. All rights reserved.
+//
+
+import Foundation
+
+struct Arcdata : Decodable {
+    var data : [ArticleAndVideoData]
+}
+
+struct ArticleAndVideoData: Identifiable, Decodable, Comparable {
+    
+    static func < (lhs: ArticleAndVideoData, rhs: ArticleAndVideoData) -> Bool {
+        lhs.attributes.released_at < rhs.attributes.released_at
+    }
+    
+    static func == (lhs: ArticleAndVideoData, rhs: ArticleAndVideoData) -> Bool {
+        lhs.attributes.released_at == rhs.attributes.released_at
+    }
+    
+    let id : String
+    let type : String
+    let attributes: ArticleAndVideoAttributee
+    let relationships: [String : Relationship]
```

Comment:
Try installing and running [SwiftLint](https://github.com/realm/SwiftLint). It will spot lots of minor formatting issues. For instance, the general practice is to not include a space before `:` between the variable name and type, the same goes for declaring dictionaries. It will also make sure your formatting is consistent across your project.